packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1595,6 +1677,30 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.services.TextInputConfiguration.enableIMEPersonalizedLearning}
   final bool enableIMEPersonalizedLearning;
 
+  /// {@template flutter.widgets.editableText.contentInsertionConfiguration}
+  /// Configuration of handler for media content inserted via the system input
+  /// method.
+  ///
+  /// By default, no handler will be created for inserted content. Set
+  /// [ContentInsertionConfiguration.onContentInserted] to provide a handler.
+  /// Additionally, set [ContentInsertionConfiguration.contentInsertionMimeTypes]
+  /// to limit the allowable mime types for inserted content.
+  ///
+  /// See [ContentInsertionConfiguration.onContentInserted] and
+  /// [ContentInsertionConfiguration.contentInsertionMimeTypes] for full examples
+  /// of using this parameter.
+  ///
+  /// Note that if [contentInsertionConfiguration] is not provided, by default
```

Comment:
Remove "Note that": https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose